### PR TITLE
DOCSP-44362 Compass 1.44.5 release notes

### PR DESCRIPTION
**Pull request by jwilson-mdb**
## DESCRIPTION
Compass 1.44.5 release notes ([gh release notes](https://github.com/mongodb-js/compass/releases/tag/v1.44.5))

## STAGING
https://deploy-preview-679--docs-compass.netlify.app/release-notes/#compass-1.44.5

## JIRA
https://jira.mongodb.org/browse/DOCSP-44362

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: snooty.toml [hunk 1]
```diff
@@ -32,7 +32,7 @@ toc_landing_pages = [
 
 [constants]
 download-page = "`downloads page <https://www.mongodb.com/try/download/compass>`__"
-current-version = "1.43.5"
+current-version = "1.44.5"
```
Comment (ajhuh-mdb) [inline snooty.toml:35]: [praise] thanks for updating this! I feel like a lot of times we forget to do this 😅

File: source/release-notes.txt [hunk 2]
```diff
@@ -10,6 +10,31 @@ Release Notes
...
+----------------
+
+*Released October 14, 2024*
+
+New Features:
+
+- Add polling for regular indexes (:issue:`COMPASS-8214`)
+- Allow editing name, color, and favorite checkbox of connected 
+  connections (:issue:`COMPASS-8160`)
+- Display banner on the connection form when
```
Comment (ajhuh-mdb) [inline source/release-notes.txt:23]: [suggestion] [style guide](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#use-a-consistent-sentence-structure) - start the sentence with the conditional clause, so users can see if the condition applies to them first.

```suggestion
- When Protect Connection String Secrets is enabled, display banner on the connection form 
```

**Review by ajhuh-mdb — APPROVED** [review]
LGTM with one non-blocking suggestion!

**Review by betsybutton — APPROVED** [review]
LGTM, thank you!

**Pull request merged**